Sources/ScipioKit/Producer/PIF/FrameworkBundleAssembler.swift

```diff
@@ -153,11 +153,96 @@ struct FrameworkBundleAssembler {
+                let destinationResourcesPath = destinationFrameworkBundlePath.appending(component: "Resources")
+                try fileSystem.copy(
+                    from: sourceResourcesPath.asURL.resolvingSymlinksInPath().absolutePath,
+                    to: destinationResourcesPath
+                )
+
+                if let resourceBundleName = sourceResourceBundlePath?.basename {
+                    let resourceBundlePath = destinationResourcesPath.appending(component: resourceBundleName)
+                    // A resource bundle of versioned bundle framework has "Contents/Resources" directory.
+                    try extractPrivacyInfoFromEmbeddedResourceBundleToFrameworkIfExists(
+                        resourceBundlePath: resourceBundlePath,
+                        relativePrivacyInfoPath: TSCRelativePath(validating: "Contents/Resources/PrivacyInfo.xcprivacy")
+                    )
+                }
+            } else {
+                try copyInfoPlist()
+                let copiedResourceBundlePath = try copyResourceBundle()
+
+                if let copiedResourceBundlePath {
+                    try extractPrivacyInfoFromEmbeddedResourceBundleToFrameworkIfExists(
+                        resourceBundlePath: copiedResourceBundlePath,
+                        relativePrivacyInfoPath: TSCRelativePath(validating: "PrivacyInfo.xcprivacy")
+                    )
+                }
+            }
+        }
 
-    private func copyResources() throws {
-        if let resourceBundlePath = frameworkComponents.resourceBundlePath {
-            let destinationPath = frameworkBundlePath.appending(component: resourceBundlePath.basename)
-            try fileSystem.copy(from: resourceBundlePath, to: destinationPath)
+        private func copyInfoPlist() throws {
+            let sourcePath = sourceFrameworkInfoPlistPath
+            let destinationPath = destinationFrameworkBundlePath.appending(component: "Info.plist")
+            try fileSystem.copy(from: sourcePath, to: destinationPath)
+        }
+
+        /// Returns the resulting, copied resource bundle path.
+        private func copyResourceBundle() throws -> TSCAbsolutePath? {
+            if let sourcePath = sourceResourceBundlePath {
+                let destinationPath = destinationFrameworkBundlePath.appending(component: sourcePath.basename)
+                try fileSystem.copy(from: sourcePath, to: destinationPath)
+                return destinationPath
+            } else {
+                return nil
+            }
+        }
+
+        /// Extracts PrivacyInfo.xcprivacy to expected location (if exists in the resource bundle).
+        ///
+        /// ref: https://developer.apple.com/documentation/bundleresources/adding-a-privacy-manifest-to-your-app-or-third-party-sdk#Add-a-privacy-manifest-to-your-framework
+        private func extractPrivacyInfoFromEmbeddedResourceBundleToFrameworkIfExists(
```

Comment:
How about converting method names to keywords to shorten the method name.

`extractPrivacyInfoIfExists(from relativePrivacyInfoPath: RelativvePath, in embeddedResourceBundlePath: AbsolutePath)`